platform-sdk/swirlds-jasperdb/src/main/java/com/swirlds/jasperdb/VirtualDataSourceJasperDB.java

```diff
@@ -152,37 +154,37 @@
      */
     private final boolean isLongKeyMode;
 
-    /** In memory off-heap store for path to disk location, this is used for both internal hashes store. */
-    private final LongList pathToDiskLocationInternalNodes;
+    /** In memory off-heap store for path to disk location. This is used for both node types. */
+    private final LongList hashPathToDiskLocation;
 
     /** In memory off-heap store for path to disk location, this is used by leave store. */
-    private final LongList pathToDiskLocationLeafNodes;
+    private final LongList leafPathToDiskLocation;
 
     /**
      * In memory off-heap store for internal node hashes. This data is never stored on disk so on load from disk, this
      * will be empty. That should cause all internal node hashes to have to be computed on the first round which will be
      * expensive.
      */
-    private final HashList internalHashStoreRam;
+    private final HashList hashStoreRam;
 
     /**
      * On disk store for internal hashes. Can be null if all hashes are being stored in ram by setting
-     * internalHashesRamToDiskThreshold to Long.MAX_VALUE.
+     * hashesRamToDiskThreshold to Long.MAX_VALUE.
      */
-    private final MemoryIndexDiskKeyValueStore<VirtualInternalRecord> internalHashStoreDisk;
+    private final MemoryIndexDiskKeyValueStore<PathHashRecord> hashStoreDisk;
```

Comment:
OK, there are three stores here. `hashStoreRam`, `hashStoreDisk`, and `pathToKeyValue`. To me, the last two are conceptually similar (both are data file stores), so I'd rename them accordingly: `pathToHash` and `pathToKeyValue`. The first one is like a cache, so it can be `pathToHashCache` or `pathToHashRam`. What do you think about it?

Reply:
Sounds good.